bpf/lib/common.h

```diff
@@ -823,6 +825,8 @@ struct lb4_backend {
 	__be16 port;		/* L4 port filter */
 	__u8 proto;		/* L4 protocol, currently not used (set to 0) */
 	__u8 flags;
+    __u8 preferred;
+    __u8 pad[2];
```

Comment:
Is it possible to integrate this into the flags attribute, so a resize is not necessary of the lb maps? (If it is needed, how is it safe during up/down-grade, could you elaborate more?)

Reply:
good point, let me explore this angle and get back later :thinking:

Reply:
thanks, I have merged preferred indicator into flags :100: